### DISCUSSION TITLE
Using the DS1307 RTC

### DISCUSSION THREAD

**Discussion by Space-Cat-Atelier** [earlephilhower/arduino-pico#2239] [Q&A]
I have a project that requires an RTC DS1307.
How would I go about using this module using Arduino-Pico?

**Comment by Andy2No** [comment]
You could refer to the documentation for the Adafruit breakout board for one:

https://learn.adafruit.com/ds1307-real-time-clock-breakout-board-kit

There's a library for Arduino provided, linked from that page (on the left menu / nav bar).  Since it's just I2C, that probably works as is.

You may also want to refer to I2C examples for the Pico using Arduino-Pico.
